src/test/java/io/stargate/sgv2/jsonapi/api/configuration/ObjectMapperConfigurationTest.java

```diff
@@ -42,6 +46,101 @@ class ObjectMapperConfigurationTest {
 
   @Inject DocumentLimitsConfig documentLimitsConfig;
 
+  @Nested
+  class unmatchedOperationCommandHandlerTest {
+    @Test
+    public void customHandlerTest() throws Exception {
+      String json =
+          """
+                  {
+                    "findOne": {
+                    }
+                  }
+                  """;
+      final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Shouldn't this test just use `objectMapper` that is injected or... ?

Reply:
deleted